common/logger.go

```diff
@@ -15,8 +16,19 @@ type Logger struct {
 	LogFile *os.File
 }
 
-func CreateLogger(logFile string) *Logger {
-	f, err := os.OpenFile(logFile, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0644)
+func CreateLogger(logPath string, logFile string) *Logger {
+	if _, err := os.Stat(logPath); os.IsNotExist(err) {
+		err := os.MkdirAll(logPath, 0755)
```

Comment:
```suggestion
		err := os.MkdirAll(logPath, 0644)
```

I think going for 644 permission is better at this point for logs. not allowing to execute log file can be great, and preventing others from changing the logs file

Reply:
Ok, just went with the default without really thinking about it